app/services/irs_attempts_api/redis_client.rb

```diff
@@ -0,0 +1,39 @@
+module IrsAttemptsApi
+  class RedisClient
+    cattr_accessor :redis_pool do
+      ConnectionPool.new(size: 10) do
+        Redis::Namespace.new(
+          'irs-attempt-api',
+          redis: Redis.new(url: IdentityConfig.store.redis_irs_attempt_api_url),
+        )
+      end
+    end
+
+    def write_event(jti, jwe)
+      redis_pool.with do |client|
+        client.set(jti, jwe)
+        client.expire(jti, IdentityConfig.store.irs_attempt_api_event_ttl_seconds)
```

Comment:
We can combine these with a `setex`, and our redis is new enough to support this, we use this elsewhere
https://redis.io/commands/setex/

```suggestion
        client.setex(jti, IdentityConfig.store.irs_attempt_api_event_ttl_seconds, jwe)
```

Reply:
great catch!